src/server/object_services/object_server.js

```diff
@@ -970,7 +970,7 @@ async function delete_multiple_objects_by_filter(req) {
         }
     }));
     const bucket_has_objects = await MDStore.instance().has_any_objects_for_bucket(bucket_id);
-    return { is_empty: !bucket_has_objects };
+    return { is_empty: !bucket_has_objects, objects_deleted: objects.length };
```

Comment:
I wonder, Do we even need the is_epmty? 
If we don't we can keep it and open a gap. (or remove it in this PR).

Reply:
I will just remove